Release: update testing instructions for 11.5 (#27027)
Automattic/jetpack: projects/plugins/jetpack/to-test.md

## Jetpack 11.5

### Before you start:

- **At any point during your testing, remember to [check your browser's JavaScript console](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis) and see if there are any errors reported by Jetpack there.**
- Use the "Debug Bar" or "Query Monitor" WordPress plugins to help make PHP notices and warnings more noticeable and report anything of note you see.

> ### Jetpack Forms block

Reviewer: ```suggestion
### Jetpack Form Block
```

> Forms block got several updates and bug fixes in this version. To test, try the following:

Reviewer: ```suggestion
The Form Block received several updates and bug fixes in this version. To test, try the following:
```

- Make sure the testing site has Jetpack connected to your account.
- Create a post and add a Form block to it. 

> - Hover your pointer between two Form child blocks (eg. Name and Email in Contact form). The Block Inserter (plus sign with Add block title) should appear, allowing you to easily insert new fields between the blocks.

Reviewer: ```suggestion
- Hover your pointer between two Form child blocks (e.g., Name and Email in Contact form). The Block Inserter (plus icon with Add block title) should appear allowing you to easily insert new fields between the blocks.
```

- Check the Dimensions section in the sidebar and make sure it already has padding setting (16 px). Try editing padding.

> - Try editing Field widths for different fields and make sure that what you see in the editor matches the frontend.

Reviewer: ```suggestion
- Try editing Field widths for different fields and make sure that what you see in the editor matches the frontend preview.
```

> - Add a Contact form and select the Form block container. Make sure the edit button doesn't show up (pen icon). All form settings such as email address and subject line should be available in the sidebar.

Reviewer: > Add a Contact form

Is this a new form block being added now, not clear among the steps to me?

Author: No, this is the old one but is getting a lot of attention in this version.

Reviewer: Sorry, I meant second form block being added in the testing steps since it starts off with "Create a post and add a Form block to it." then later on the "Add a Contact form".

If it's a second form block being added, might help to break things up a bit.

> - Add a Contact form and select the Form block container. Make sure the edit button doesn't show up (pen icon). All form settings such as email address and subject line should be available in the sidebar.

Author: Does this sound better @samiff?

```suggestion
- If you tested using a different type of form, try adding a Contact form and select the Form block container. Make sure the edit button doesn't show up (pen icon). All form settings such as email address and subject line should be available in the sidebar.
```

- Lose and regain focus on the Form block by clicking inside of it. Clicking the empty space between child elements should select the main Form block.
- See if form related blocks show up in the block library when trying to insert a new child Form block. Try setting the focus on a child block and adding a block. The section with the Form blocks should have a title "FORMS".

> - Add a Contact Form and add a Consent block to it (any type). Configure the Email address to send to to your email. Pick "Show summary of submitted fields" as On submit action. Submit the form on the frontend and see that the "Consent: Yes" is visible on the Success page.

Reviewer: > Configure the Email address to send to to your email.

Might clarify that the notification setting is being edited here.

> - Add a Contact Form and add a Consent block to it (any type). Configure the Email address to send to to your email. Pick "Show summary of submitted fields" as On submit action. Submit the form on the frontend and see that the "Consent: Yes" is visible on the Success page.

Author: Better? 

```suggestion
- Add a Contact Form and add a Consent block to it (any type). Configure the form so the "Email address to send to" is set to your email. Pick "Show summary of submitted fields" as On submit action. Submit the form on the frontend and see that the "Consent: Yes" is visible on the Success page.
```

> ### Jetpack Social

Reviewer 2: Ah this confused me, I wonder if the heading should be Jetpack Subscriptions instead since it affects the subscriptions block / subscriber counts?
```suggestion
### Jetpack Subscription block
```

> Jetpack Social connections are no longer counted in the Subscriptions block. To test, try the following:

Reviewer: > Jetpack Social connections are no longer counted in the Subscriptions block

To the average person, I think this could be a tough statement to decipher. We should be very clear about what is changing here. Also reference internal: p58i-dod-p2

> Jetpack Social connections are no longer counted in the Subscriptions block. To test, try the following:

Author: This one is confusing. How about this?

```suggestion
Subscriptions block used to display total number of subscribers that included users from social networks. We are changing the Subscriptions block to display only email subscribers and WordPress.com followers. To test, try the following:
```

> - On a Jetpack connected site, enable Jetpack Social and connect to Twitter. Jou should have at least one follower.

Reviewer: ```suggestion
- On a Jetpack connected site, enable Jetpack Social and connect to Twitter. You should have at least one follower.
```

> - Create a new post and add the subscription block.

Reviewer: ```suggestion
- Create a new post and add the Subscription block.
```
